x-pack/test/cloud_security_posture_functional/page_objects/findings_page.ts

```diff
@@ -106,7 +106,7 @@ export function FindingsPageProvider({ getService, getPageObjects }: FtrProvider
 
   const createNotInstalledObject = (notInstalledSubject: string) => ({
     getElement() {
-      return testSubjects.find(notInstalledSubject);
+      return testSubjects.find(notInstalledSubject, 15 * 1000);
```

Comment:
have you tried to use `await pageObjects.header.waitUntilLoadingHasFinished();` instead?

Reply:
@JordanSh yes, `await PageObjects.header.waitUntilLoadingHasFinished();` is already there before `getElement()` in the test itself, eg. in `clicking on the `No integrations installed` prompt action button - `install CNVM`: navigates to the CNVM integration installation page` test. For some reason our page wrapper is still in the loading state when I look at thw failures